core/src/main/resources/application.conf

```diff
@@ -255,38 +255,38 @@ backend {
     //        }
     //      }
     //    }
-    }
   }
+}
 
-  services {
-    KeyValue {
-      class = "cromwell.services.KeyValueServiceActor"
-    },
-    MetadataService {
-      class = "cromwell.services.EngineMetadataServiceActor"
-    }
+services {
+  KeyValue {
+    class = "cromwell.services.KeyValueServiceActor"
+  },
+  MetadataService {
+    class = "cromwell.services.EngineMetadataServiceActor"
   }
+}
 
-  database {
-    config = main.hsqldb
+database {
+  config = main.hsqldb
 
-    main {
-      hsqldb {
-        db.url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
-        db.driver = "org.hsqldb.jdbcDriver"
-        db.connectionTimeout = 1000 // NOTE: 1000ms (the default) is ok for a small hsqldb, but often too short for mysql
-        driver = "slick.driver.HsqldbDriver$"
-      }
+  main {
+    hsqldb {
+      db.url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
+      db.driver = "org.hsqldb.jdbcDriver"
+      db.connectionTimeout = 1000 // NOTE: 1000ms (the default) is ok for a small hsqldb, but often too short for mysql
```

Comment:
I think this comment is now out of date - there is a separate stanza for mysql now